test/experimental/test_vocab.py

```diff
@@ -17,21 +17,25 @@ def tearDown(self):
         torch._C._jit_clear_class_registry()
         torch.jit._recursive.concrete_type_store = torch.jit._recursive.ConcreteTypeStore()
 
-    def test_has_unk(self):
+    # we separate out these errors because Windows runs into seg faults when propagating
+    # exceptions from C++ using pybind11
+    @unittest.skipIf(platform.system() == "Windows", "Test is known to fail on Windows.")
+    def test_has_no_unk(self):
         c = OrderedDict()
         v = vocab(c)
+        self.assertEqual(v.get_default_index(), -1)
```

Comment:
I think it's better if this were to return "None". You should be able to do this easily by using c10::optional<int64_t> instead of int64_t in the C++ code.